airbyte-cdk/python/airbyte_cdk/sources/file_based/file_types/parquet_parser.py

```diff
@@ -66,8 +66,8 @@ def parse_records(
             raise ConfigValidationError(FileBasedSourceError.CONFIG_VALIDATION_ERROR)
 
         line_no = 0
-        try:
-            with stream_reader.open_file(file, self.file_read_mode, self.ENCODING, logger) as fp:
+        with stream_reader.open_file(file, self.file_read_mode, self.ENCODING, logger) as fp:
```

Comment:
do we need to update the other parsers too?

Reply:
nope, others are fine